v0/destinations/facebook_pixel/transform.js

```diff
@@ -589,14 +589,18 @@ const processEvent = (message, destination) => {
     eventsToEvents = getValidDynamicFormConfig(
       destination.Config.eventsToEvents,
       "from",
-      "to"
+      "to",
+      destination.Name,
```

Comment:
We need this to be destType, if not available form config lets pass it as an argument which is hardcoded
```suggestion
      "to",
      "FB_PIXEL"
```